src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,35 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export function PostInfo({ post: { title, user, body, comments } }) {
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
+          user &&
+          <UserInfo user={user} />
+        </p>
```

Comment:
1. it's better to wrap with `div` (since your UserInfo component can be changed and not all tags can be included into `p`)
2. use `span` tag
3. use correct semantic expression for `UserInfo` (at screen you can see how it looks like now)
<img width="171" alt="Screenshot 2023-12-05 at 18 03 21" src="https://github.com/mate-academy/react_static-list-of-posts-js/assets/117587463/af6ffc78-a562-4bcb-b9ec-1cfdc75e9120">

```suggestion
        <div>
          <span> Posted by  </span>
          {user && <UserInfo user={user} />}
        </p>
```